src/FSharpVSPowerTools.Core/Symbols/HighlightUsageInFile.fs

```diff
@@ -0,0 +1,41 @@
+namespace FSharpPowerTools.Core
+open FSharpVSPowerTools
+module Symbols = 
+    open System.IO
+    
+    open Microsoft.FSharp.Compiler.SourceCodeServices
+    
+    let filterSymbolUsesDuplicates (uses: FSharpSymbolUse []) =
+        uses
+        |> Seq.map (fun symbolUse -> (symbolUse.FileName, symbolUse))
+        |> Seq.groupBy (fst >> Path.GetFullPathSafe)
+        |> Seq.collect (fun (_, symbolUses) -> 
+            symbolUses 
+            |> Seq.map snd 
+            |> Seq.distinctBy (fun s -> s.RangeAlternate))
+        |> Seq.toArray
+
+type FileName = string
+type GetCheckResults = FileName -> Async<ParseAndCheckResults option>
+type FCSZeroBasedRange = int * int * int * int
+
+type CurrentLine = {Line: string; File: FileName; Range: FCSZeroBasedRange}
+    with
```

Comment:
This doesn't seem to work with records, I assume those are seen as (intrinsic) extension members.